README.md

```diff
@@ -21,41 +19,29 @@
  You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
+## Instructions
 
-
-#################
-  INSTRUCTIONS
-#################
-
-OpenShot Video Editor 2.0 supports Linux, Mac, and Windows. But due to 
+OpenShot Video Editor 2.x supports Linux, Mac, and Windows. But due to 
 the many dependencies required to run OpenShot, we recommend using an 
 installer for your operating system. Please visit 
 http://www.openshot.org/download/ to find your correct installer.
 
+## Documentation
 
-
-#################
- DOCUMENTATION
-#################
-
-Documentation for OpenShot 2.0 can be generated with Doxygen, a popular
+Documentation for OpenShot 2.x can be generated with Doxygen, a popular
 command-line application for scanning source code and generating readable
 documentation. OpenShot contains a Doxygen input file (Doxyfile.in), but
 you must first install the following Python filter for Doxygen: doxypy.
 Then, run the following command:
 
 $ doxygen Doxyfile.in
```

Comment:
All of the example command lines need to be enclosed in code blocks, which is probably most readable (in the non-formatted source) if it's done by indenting each line four spaces.